module-2/backend/week_two.md

```diff
@@ -4,34 +4,25 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 
 Note: When you're done, submit a PR. 
 
-1. At a high level, what is ActiveRecord? What does it do/allow you to do?
-2. What kind of methods are `belongs_to`, and `has_many`? (i.e. class or instance) Give an example.
-3. What do they allow you to do?
-4. What's the difference between agile workflow and waterfall method?
-5. What is the difference between `#new` and `#create`?
-6. At a basic level, what does cURL allow you to do?
-7. In a database that's holding students and teachers, what will be the relationship between students and teachers? Draw the schema diagram.
-8. Define foreign key, primary key, and schema.
-9. Describe the relationship between a foreign key on one table and a primary key on another table.
-10. What are the parts of an HTTP response?
-11. Describe some techniques to make our Sinatra code more DRY. Give an example of when you would use these techniques.
+1. At a high level, what is ActiveRecord? What does it do/allow you to do? It's a database management system that allows us to interact with a database through code. I think it's an object relational mapping system... which means that it "wraps" a database in a relational system that we can then acess in our code without having to write raw SQL queries. Something like that. 
+2. What kind of methods are `belongs_to`, and `has_many`? (i.e. class or instance) Give an example. These are class methods.
```

Comment:
Looks like you're using a stale version. If you're working on your forked version, be sure to add this repo as an `upstream` repo ([instructions here](https://help.github.com/articles/configuring-a-remote-for-a-fork/)), and then pull the most recent version before you start answering questions.